benchmark/micro/compression/zstd/zstd_read.benchmark

```diff
@@ -0,0 +1,17 @@
+# name: benchmark/micro/compression/zstd/zstd_read.benchmark
+# description: ZSTD decompression speed of relatively big (8000 byte) strings
+# group: [zstd]
+
+name ZSTD Scan
+group zstd
+storage persistent
+
+load
+DROP TABLE IF EXISTS zstd_strings;
+PRAGMA force_compression='zstd';
+set variable my_string = (list_reduce([chr(((i % 26) + ord('a'))::INTEGER) for i in range(8000)], (x, y) -> concat(x, y)));
+create table zstd_strings as select getvariable('my_string') as data from range(10_000_000) tbl(i);
+checkpoint;
+
+run
+select avg(strlen(data)) from zstd_strings;
```

Comment:
Can we also add a compression benchmark? And post results of the zstd compression vs other compression algorithms on different data sets